file4.py

```diff
@@ -0,0 +1,5 @@
+#printing a before declaring
+print(a)
+
+a=10
```

Comment:
```suggestion
print(10) 
```
directly, no need to use additional variable